container-toolkit/install-guide.md

```diff
@@ -56,7 +56,12 @@ Alternatively, you can install the driver by [downloading](https://www.nvidia.co
 1. Install the NVIDIA Container Toolkit packages:
 
    ```console
-   $ sudo apt-get install -y nvidia-container-toolkit
+   $ NVIDIA_CONTAINER_TOOLKIT_VERSION=1.17.7-1 \
```

Comment:
should this be 1.17.7-1 for every release? Should we make it a variable so that the version numbers updates?

Reply:
Maybe we can update this to 1.17.8-1 in this PR and then look to parameterize it in a follow-up?

Reply:
This was moved to #190